app/models/spree/order_decorator.rb

```diff
@@ -378,6 +378,13 @@ def address_from_distributor
     address
   end
 
+  # Update attributes of a record in the database without callbacks, validations etc.
+  #   This was originally a ext to ActiveRecord in Spree but only used for Spree::Order
+  def update_attributes_without_callbacks(attributes)
```

Comment:
And since Rails 3.1 there is `update_columns` which does the same.
https://apidock.com/rails/ActiveRecord/Persistence/update_columns